### DISCUSSION TITLE
How to change the TSR in the Gaussian Model

### DISCUSSION THREAD

**Discussion by paulf81** [NatLabRockies/floris#180] [Q&A] [answered]
Dear all,

I am wondering to analyzing the wake velocity deficit by operating in a different TSR in the upstream turbine. I mean operating in non-optimum TSR and I would like to use FLORIS for that. I’m using the gaussian model and the example that we provide the Cp and Ct as functions of the wind speed. How could I perform this analysis in FLORIS by changing the TSR?

Regards

**Comment by paulf81** [comment, marked as answer]
TSR is not directly included in the Cp/Ct model in FLORIS currently.  If you change TSR, you will need to directly access the source code to update the Cp/Ct based on TSR.  This is not a heavy lift, but will need to be done to get accurate results.

**Reply by tuyu99** [reply]
Dear paulf81,
Your question and answer is very reasonable! Recently, I have the same problem. **In two in-line turbines, change the TSR of upstream turbine and get the Cp of two turbines.**
After searching for the method, I found the same conclusion as yours. **"TSR is not directly included in the Cp/Ct model in FLORIS currently. If you change TSR, you will need to directly access the source code to update the Cp/Ct based on TSR. This is not a heavy lift, but will need to be done to get accurate results."** 
Additionally, NREL suggests use OpenFAST to get the wind speed-Cp-Ct table under the openrational condition we want. I installed the OpenFAST and tried to get the data of table. Firstlly, I think I should figure out the set-up of OpnFAST. So I tried to calculate the given data of table under _TSR=8, Wind speed = 8 m/s_ and I failed.
I was wondering how you fix the problem at last ?
Best Regards
